alerting/src/main/java/org/hawkular/metrics/alerting/InsertedDataSubscriber.java

```diff
@@ -129,10 +132,33 @@ private void publishNumeric(Metric<? extends Number> numeric) {
         }
     }
 
+    private String getDataId(MetricId<? extends Number> numericId) {
```

Comment:
Wouldn't an enum make more sense here? Or a map created in the beginning of type <MetricId, String> ?